app/wrappers/Applicant.js

```diff
@@ -10,6 +10,15 @@ class Applicant {
     applicantHasDeclared() {
         return !isEmpty(this.formdata.declaration) && this.formdata.declaration.declarationCheckbox === 'true';
     }
+
+    applicantIsChild() {
+        return this.formdata.applicant && (this.formdata.applicant.relationshipToDeceased === 'optionChild' || this.formdata.applicant.relationshipToDeceased === 'optionAdoptedChild');
+    }
+
+    spouseIsRenouncing() {
+        return this.formdata.applicant && this.formdata.applicant.spouseNotApplyingReason === 'optionRenouncing';
+    }
+
```

Comment:
I would have named these bit like isSpouseRenouncing, isApplicantChild then obvious you are expecting true or false without going off to look at it

Reply:
done